09_json/runnerdave/data.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
I would recommend you put your data files under a directory `$PROJECT_ROOT/data` or perhaps `$PROJECT_ROOT/puppydata`.
eg:
`runnerdave/puppydata/data.json`
`runnerdave/puppydata/invalid-data.json`

Reply:
done